x-pack/plugins/fleet/common/openapi/paths/outputs@{output_id}.yaml

```diff
@@ -53,42 +48,14 @@ put:
     content:
       application/json:
         schema:
-          type: object
-          properties:
-            name:
-              type: string
-            type:
-              type: string
-              enum: ['elasticsearch']
-            is_default:
-              type: boolean
-            is_default_monitoring:
-              type: boolean
-            hosts:
-              type: array
-              items:
-                type: string
-            ca_sha256:
-              type: string
-            ca_trusted_fingerprint:
-              type: string
-            config_yaml:
-              type: string
-          required:
-            - name
-            - type
+          $ref: ../components/schemas/output.yaml
```

Comment:
I am wondering if we should have a different type for the PUT request as most the Fleet API are in reality PATCH API and fields are optional in that case

Reply:
Yes, you are right, making two separate schemas makes sense because now we do not reflect POST requirements properly. Thanks :)

Reply:
Addressed in a74dc5815f26c1d0fd9051f699cba32ad18ba8ad